src/events/processor/find-features/index.js

```diff
@@ -104,6 +104,16 @@ function matchFeature(props, featuresData) {
   return undefined;
 }
 
+/**
+ * @param {string?} locationItem
+ * @param {{ properties: { name: string?; name_en: string?; region: string?; }; }} feature
+ */
+const doesLocationMatchFeature = (locationItem, feature) =>
+  locationItem &&
+  [feature.properties.name, feature.properties.name_en, feature.properties.region].some(
+    toMatch => toMatch === locationItem
+  );
```

Comment:
```suggestion
function doesLocationMatchFeature(locationItem, feature) {
  return locationItem &&
  [feature.properties.name, feature.properties.name_en, feature.properties.region].some(
    toMatch => toMatch === locationItem
  );
}
```

Reply:
Why?
Assuming a good reason, we can enforce via linting if we want to.